app/assets/javascripts/header.js

```diff
@@ -94,4 +68,36 @@ function setHeight() {
   $('#header_space').css({"height": the_height});
 }
 
+function expandButton(event) {
+  hideSmallTopNav();
+  console.log(event.data);
+  if (event.data.value % 2 == 0) {
+    showExpandMe();
+  } else {
+    hideExpandMe();
+  }
+  setHeight();
+  event.data.value++;
+}
+
+function headerMouseLeave() {
+  if ($('#expand_moment').length && $('#expand_moment')[0].classList.contains('display_block')) {
```

Comment:
`$('#expand_moment')[0].classList.contains('display_block')` is quite the "mouthful" – it's hard to understand what the code is doing without reading it closely. What do you think about extracting this expression into it's own function? The name of the function will help with explaining the intent of the code!